etc/openqa/openqa.ini

```diff
@@ -136,3 +136,10 @@ blocklist = job_grab job_done
 # Specify the number of screenshot ID ranges (with a size as configured by screenshot_cleanup_batch_size) to process in a single Minion
 # job (reduce to avoid Minion jobs from running very long and possibly being interrupted, increase to reduce the number of Minion jobs)
 #screenshot_cleanup_batches_per_minion_job = 450
+
+[job_settings_ui]
+# Specify the keys of directories which contain test suite settings
+# To enable settings rendered as links and add a key=value to the `etc/openqa/settings_ui_links.ini`
```

Comment:
If this is a consecutive sentence please continue with lower-case in the second line. To make the benefit obvious I would turn it around: "To render individual paths in job settings as links …"

Reply:
done